scripts/app.js

```diff
@@ -1 +1,30 @@
 'use strict';
+var slideshow = [];
```

Comment:
Is this supposed to be for your slideshow or for the HW requirement of adding your portfolio data to your page?